perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -198,8 +221,29 @@ def PrepareVMEnvironment(self):
       if self.is_static:
         self.SnapshotPackages()
       self.SetupPackageManager()
+    self.ConfigureVMKernel()
     self.BurnCpu()
 
+  def ConfigureVMKernel(self):
+    """Apply {procfs,sysfs}_config to VM."""
+
+    path_components = None
+
+    def ConfigureSubtree(subtree):
+      for name, value in subtree.iteritems():
+        path_components.append(name)
+        if isinstance(value, basestring):
```

Comment:
My original plan was that users would explicitly enter a string value, like {"e": "4"} in your example. That would make the value they enter fit with what we're actually going to do, which is write out the value as a string.

However, putting quotes around integers is annoying. I will implement the non-dict check you suggested and `str`-convert the output to strings. Thanks!